docs/en/operations/settings/settings.md

```diff
@@ -250,6 +250,28 @@ Sets default strictness for [JOIN clauses](../../query_language/select.md#select
 
 **Default value**: `ALL`
 
+## join_any_take_last_row {#settings-join_any_take_last_row}
+
+Changes behavior of ClickHouse for join operations with `ANY` strictness.
+
+!!! note "Attention"
+    This setting applies only for the [Join](../table_engines/join.md) table engine.
+
+Possible values:
+
+- 0 — ClickHouse uses standard logic.
+
+    If the right table has several matching rows, only the first one found is joined.
```

Comment:
"several" means ~3-5, for example it'd be better with "more than one matching row"